### ISSUE TITLE
Accessibility issue, Required ARIA child role not present: : textbox

### ISSUE THREAD

**Issue by ElenaShevchenko** [jackocnr/intl-tel-input#1174]
### Steps to reproduce
1.  Set "separateDialCode" option to true 

### Expected behaviour
<div class="iti__selected-flag" role="combobox" aria-controls="iti-0__country-listbox" aria-owns="iti-0__country-listbox" aria-expanded="false">....
This div has role="combobox" , according to accessibility requests "Authors MUST ensure an element with role combobox contains or owns a text input element with role textbox or searchbox"  https://www.w3.org/TR/wai-aria-1.1/#combobox
So  role="textbox" should be added to <div class="iti__selected-dial-code">...</div>

### Actual behaviour
There is no role="textbox" for child element

### Initialisation options
separateDialCode = true

**Comment by ElenaShevchenko** [comment]
aria-label, aria-labelledby, title should be provided as well for textbox

**Comment by jackocnr** [comment]
Thanks for raising this issue. I'm afraid I don't have time to work on this myself right now, but would welcome a pull request :)

**Comment by ElenaShevchenko** [comment]
@jackocnr could you please add me to contributors list, I try to push changes but I have no permission

**Comment by jackocnr** [comment]
Please see Contributing guide: https://github.com/jackocnr/intl-tel-input/blob/master/.github/CONTRIBUTING.md

**Comment by jackocnr** [comment]
Responded in the PR, but I don't think this is required in this case: https://github.com/jackocnr/intl-tel-input/pull/1175. Thanks.
